engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/UpdateByWindowRollingTime.java

```diff
@@ -168,100 +152,105 @@ private static WritableRowSet computeInfluencerRowsTime(final UpdateByWindowTime
     public void computeAffectedRowsAndOperators(UpdateByWindowBucketContext context, @NotNull TableUpdate upstream) {
         UpdateByWindowTimeBucketContext ctx = (UpdateByWindowTimeBucketContext) context;
 
-        if (upstream.empty() || ctx.sourceRowSet.isEmpty()) {
-            // No further work will be done on this context
-            finalizeWindowBucket(context);
-            return;
-        }
+        try (final ChunkSource.GetContext tsContext = ctx.timestampColumnSource.makeGetContext(ctx.workingChunkSize)) {
```

Comment:
This is the important line.  `Try-with-resources` here and pass the context to the calling functions instead of keeping as a member.

Reply:
Github is having a problem with matching the lines, even though they are unchanged. Sorry for the confusion